tox.ini

```
@@ -1,9 +1,9 @@
 [tox]
 envlist =
     # NOTE: keep this in sync with the env list in .github/workflows/ci.yml.
-    py{38,39,310,311,312,py3}-pip{previous,latest,main}-coverage
-    pip{previous,latest,main}-coverage
-    pip{previous,latest,main}
+    py{38,39,310,311,312,py3}-pip{supported,lowest,latest,main}-coverage
+    pip{supported,lowest,latest,main}-coverage
+    pip{supported,lowest,latest,main}
```

```suggestion
    py{38,39,310,311,312,py3}-pip-{supported,lowest,latest,main}-coverage
    pip-{supported,lowest,latest,main}-coverage
    pip-{supported,lowest,latest,main}
```
Could we add dashes (and of course elsewhere where referenced) to improve readability?

---

Later. See the other comment.